src/NLog.Targets.Syslog/Policies/ReplaceKnownValuePolicy.cs

```diff
@@ -31,7 +33,8 @@ public string Apply(string s)
                 return s;
 
             var replaced = Regex.Replace(s, searchFor, replaceWith);
-            InternalLogger.Trace("[Syslog] Replaced '{0}' (if found) with '{1}' given known value '{2}': '{3}'", searchFor, replaceWith, s, replaced);
+            if (!ReferenceEquals(replaced, s))
+                InternalLogger.Trace("[Syslog] Replaced {0} given computed value '{1}': '{2}'", applytracemessage, s, replaced);
```

Comment:
`String.Format` allocates only one string filling all placeholders with values:
is the optimization focused on avoiding the substitution of two parameters on trace?